imgProc_sm.m

```diff
@@ -553,6 +562,30 @@ function imgProc_sm(infile, outfile, probename, n, nEvery, projectname, varargin
                     particle_partNum(kk)=bin2dec([dec2bin(data(start-1,7),8),dec2bin(data(start-1,8),8)]);
 
                     time_in_seconds(kk) = part_hour(kk) * 3600 + part_min(kk) * 60 + part_sec(kk) + part_mil(kk)/1000 + part_micro(kk);
+                    
+                    % Deprecated - can be used to apply project/date specific time offset corrections
+                    %{
+                    if (exist('TimeOffsetSec') && TimeOffsetSec ~= 0 && (handles.month ~= 7 && handles.day ~= 1))
+                    	if (firstTime == 1 && kk == 1)
+                    		fprintf('Time correction applied to %s: %d seconds\n',probename,TimeOffsetSec);
+                    	end
+						%fprintf('Time in seconds before correction: %d\n',time_in_seconds(kk));
+						%fprintf('Particle hour before correction: %d\n',part_hour(kk));
+						%fprintf('Particle minute before correction: %d\n',part_min(kk));
+						%fprintf('Particle sec before correction: %d\n',part_sec(kk));
+						
+						time_in_seconds(kk) = time_in_seconds(kk) + TimeOffsetSec;
```

Comment:
I can test these corrections on GCPEx CIP data when I get around to it next week, and see if these need to be implemented for CIP/PIP data for any project type.